src/Network/VCR/Middleware.hs

```diff
@@ -75,27 +76,29 @@ findAnyResponse cassetteIORef savedRequest = do
     note ("The request: " <> tshow savedRequest <> " is not recorded! Ignored headers: " <> tshow ignoredHeaders) $
     find (\c -> request c == savedRequest) apiCalls
 
--- | A policy for obtaining response which expects the request to be issued in the order they were recorded.
+
+-- | A policy for obtaining r esponse which expects the request to be issued in the order they were recorded.
 consumeRequestsInOrder :: FindResponse
 consumeRequestsInOrder cassetteIORef savedRequest = do
-  cassette@Cassette { apiCalls, ignoredHeaders } <- readIORef cassetteIORef
+  cassette@Cassette { apiCalls, ignoredHeaders, ignoredBodyFields } <- readIORef cassetteIORef
```

Comment:
We do, although I can see now that the code is a little bit confusing:
we actually modify the `savedRequest` during the build phase, in `buildRequest`, so no matter what matching method we use, it will run on the modified request.

Btw. @kozak the name `savedRequest` is very misleading (to me, at least) -- my intuition is that the saved request is the one that we have in the cassette file, while here it's more of `currentRequest` or something like that. WDYT?

Reply:
Ok makes sense. I agree that the names are confusing - `currentRequest` might be better :)